packages/browser/package.json

```diff
@@ -62,6 +62,6 @@
     "preinstall": "npx only-allow pnpm"
   },
   "dependencies": {
-    "@napi-rs/wasm-runtime": "^0.2.8"
+    "@napi-rs/wasm-runtime": "^0.2.10"
```

Comment:
Does `lint-knip` throw a warning for this? @webpro 
https://github.com/rolldown/rolldown/blob/bf212da494f45706f423228d7b01185dccc86d89/package.json#L11